package.json

```diff
@@ -16,12 +17,15 @@
     "@babel/runtime": "^7.6.0",
     "@hapi/hapi": "^18.4.0",
     "async": "^3.1.0",
-    "mysql": "^2.17.1"
+    "config": "^3.2.2",
+    "mysql": "^2.17.1",
+    "open": "^6.4.0"
   },
   "devDependencies": {
     "@babel/cli": "^7.6.0",
     "@babel/core": "^7.6.0",
     "@babel/plugin-transform-runtime": "^7.6.0",
-    "@babel/polyfill": "^7.6.0"
+    "@babel/polyfill": "^7.6.0",
+    "cross-env": "^6.0.0"
```

Comment:
`cross-env` is used in places where we want to change `NODE-ENV` kind of for each script we want to run.

Documentation: https://www.npmjs.com/package/cross-env